packages/lib/env.ts

```diff
@@ -167,6 +168,7 @@ export const env = createEnv({
     S3_REGION: process.env.S3_REGION,
     S3_SECRET_KEY: process.env.S3_SECRET_KEY,
     S3_ENDPOINT_URL: process.env.S3_ENDPOINT_URL,
+    S3_FORCE_PATH_STYLE: process.env.S3_FORCE_PATH_STYLE,
```

Comment:
Please fix linting issue "S3_FORCE_PATH_STYLE is not listed as a dependency in turbo.json"